.github/workflows/lint-build.yml

```diff
@@ -0,0 +1,27 @@
+name: lint & build
+
+on: push
+
+jobs:
+  lint-build:
+    runs-on: ubuntu-latest
+    env:
+      BABAJKA_GOOGLE_API_KEY: ${{ secrets.BABAJKA_GOOGLE_API_KEY }}
```

Comment:
btw why do we need in CI ? to fetch data on postinstall? do we really want to fetch it here?